src/layouts/EditPage.jsx

```diff
@@ -51,8 +53,17 @@ const EditPage = ({ match }) => {
   const [hasChanges, setHasChanges] = useState(false)
   const [isContentViolation, setIsContentViolation] = useState(false)
   const [isXSSViolation, setIsXSSViolation] = useState(false)
-  const [showXSSWarning, setShowXSSWarning] = useState(false)
-  const [showOverwriteWarning, setShowOverwriteWarning] = useState(false)
+
+  const {
+    isOpen: isXSSOpen,
+    onOpen: onXSSOpen,
+    onClose: onXSSClose,
+  } = useDisclosure()
+  const {
+    isOpen: isOverwriteOpen,
+    onOpen: onOverwriteOpen,
+    onClose: onOverwriteClose,
```

Comment:
add `modal`; we should also make the variable name for the XSS modal warning more descriptive - perhaps something like `XSSWarningModal`?

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/0721a811504d1b7b38d4d6eb61364e11db356000